docs/user/ppl/functions/ip.rst

```diff
@@ -0,0 +1,32 @@
+====================
```

Comment:
You'll need to edit category.json in doc-test to get this example to be validated.

Reply:
Thanks. Done ✅